portfolio/src/main/webapp/script.js

```diff
@@ -137,8 +148,34 @@ function visualizeMap() {
       mapTypeControlOptions: {
         mapTypeIds: ['hybrid', 'styles_map']
       }
-    });
+  });
+  
+  //Marker positioned at my house. 
+  var markerHome = new google.maps.Marker({
+    position: myHome,
+    map,
+    title: 'My house!'
+  });
 
+  // Marker positioned at Maracaibo Lake.
+  var markerLake = new google.maps.Marker({
```

Comment:
Ditto

Reply:
Perfect, thanks for the advice will keep it in mind for the future :) Fixed the issues, should be okay now!